src/components/media-loader.js

```diff
@@ -356,6 +357,11 @@ AFRAME.registerComponent("media-loader", {
         this.showLoaderTimeout = setTimeout(this.showLoader, 100);
       }
 
+      //check if url is an anchor hash e.g. #Spawn_Point_1
+      if (src.charAt(0) === "#") {
+        src = this.data.src = `${window.location.origin}${window.location.pathname}${src}`;
```

Comment:
Might want to include `${window.location.search}` after `pathname` (these are the query string parameters, if any)